commons/src/main/java/io/aiven/kafka/tieredstorage/commons/storage/s3/S3StorageConfig.java

```diff
@@ -43,6 +43,10 @@ public class S3StorageConfig extends AbstractConfig {
     public static final String S3_REGION_CONFIG = "s3.region";
     static final String S3_REGION_DEFAULT = Regions.DEFAULT_REGION.getName();
     private static final String S3_REGION_DOC = "AWS region where S3 bucket is placed";
+    private static final String S3_PATH_STYLE_ENABLED_CONFIG = "s3.path.style.access.enabled";
+    static final boolean S3_PATH_STYLE_ENABLED_DEFAULT = false;
```

Comment:
The AWS SDK has three settings: `true`, `false`, and `null`, the latter being the default (meaning auto-detect). We should follow this.

Reply:
Great catch! Adding null as default value and updating the docs/tests.